src/gretel_client/client.py

```diff
@@ -45,6 +45,7 @@
 PROMPT = "prompt"
 PROMPT_ALWAYS = "prompt_always"
 DEFAULT_API_ENV_KEY = "GRETEL_API_KEY"
+DEFAULT_PROJECT_URI = "GRETEL_PROJECT_URI"
```

Comment:
Already went with GRETEL_URI other places, any need to change it?

Reply:
Nope, good catch. Switched to `GRETEL_URI`.